src/System.Memory/src/System.Memory.csproj

```diff
@@ -46,10 +53,11 @@
     <Reference Include="System.Reflection" />
     <Reference Include="System.Resources.ResourceManager" />
     <Reference Include="System.Runtime" />
+    <Reference Include="System.Runtime.Extensions" />
     <Reference Include="System.Runtime.InteropServices" />
     <Reference Condition="'$(TargetGroup)' != 'netstandard1.1'" Include="System.Numerics.Vectors" />
     <Reference Condition="'$(TargetGroup)' != 'netstandard1.1'" Include="System.Runtime.CompilerServices.Unsafe" />
-    <ProjectReference Condition="'$(TargetGroup)' == 'netstandard1.1'" Include="..\..\System.Runtime.CompilerServices.Unsafe\ref\System.Runtime.CompilerServices.Unsafe.csproj" />
```

Comment:
Why did you have to change from ref to src here?

Reply:
Also why do we need this to be a ProjectReference at all? A normal reference should work.

Reply:
From @safern, https://github.com/dotnet/corefx/pull/24400#discussion_r142562977
> That is correct, the only thing that is not correct is that all the ProjectReference that we add in a src project has to be to src projects not reference assembly.
So the reference below has to be a reference to the src project as well.

For netstandard, there is a normal reference.
But for netstandard1.1, we need a project reference since the Unsafe class was added in netstandard1.6.
https://apisof.net/catalog/System.Runtime.CompilerServices.Unsafe
![image](https://user-images.githubusercontent.com/6527137/31523209-fc5d0b78-af66-11e7-949f-aad8830cdf7d.png)

Reply:
>But for netstandard1.1, we need a project reference since the Unsafe class was added in netstandard1.6.

I'm not sure I understand this. If we are building this library for netstandard1.1 then it cannot depend on a netstanard1.6 library. You might be getting away with this because of the ProjectReference but I still believe this should just be a Reference.